### ISSUE TITLE
scaleway provider

### ISSUE THREAD

**Issue by webroboteu** [hashicorp/terraform#9235]
Hello.

to access the the scaleway apis now ewe need to send the user token and the organization id and not the access token.
I'm doing something wrong or is in fact the new scenario

Roger

**Comment by nicolai86** [comment]
Hey @webroboteu,

you need two things for the provider to work:

- the user access token. Once logged in to the scaleway UI you can manage access tokens in your settings under credentials: https://cloud.scaleway.com/#/credentials
- your organization ID. Since this is not listed on the scaleway UI, the most reliable way is to use the [scaleway CLI](https://github.com/scaleway/scaleway-cli). Once you login via the CLI (`scw login`) your organization ID is written to `~/.scwrc`. E.g. `cat ~/.scwrc | jq -r '.organization'`

I guess we should add this information to the provider page.

**Comment by far-blue** [comment]
I've just started playing with the scaleway provider and found the following that might help you:
* The `organisation` value needed by the provider is now listed in the account credentials section (https://cloud.scaleway.com/#/credentials) as `access key` just above the table of access tokens.
* The `access_key` value needed by the provider is actually a token so just create a new token in the credentials section of your account and give it a helpful label for future reference.
* Server image values are not the same as the name given to the image when you create a server via the web control panel. I've yet to find a way to show the image id values via the web interface so I had to resort to using the Rest API. For this I used Postman (a great Rest API client), set a header of X-Auth-Token to the token I mentioned in the previous bullet point, set a header of Content-Type to `application/json` and made a GET request to https://api.scaleway.com/images which listed all the images available to me. Hunt around in there for the image you want - noting the architecture is correct - and then use the `id` value in the `image` key for the server.
